src/nspawn/nspawn-seccomp.c

```diff
@@ -171,6 +171,13 @@ static int add_syscall_filters(
                 log_warning_errno(r, "Failed to add rule for @known set on %s, ignoring: %m",
                                   seccomp_arch_to_string(arch));
 
+        /* We have a large filter here, so let's turn on the binary tree mode if possible. */
```

Comment:
doesn't matter, but i'd move this into the `#if` block, since this is not a comment about the ifdeffery but about the function call